pkg/utils/apiutil/apiutil.go

```diff
@@ -311,6 +311,38 @@ func CollectEscapeStringOption(option string, input map[string]any, collectors .
 	return errs.ErrOptionNotExist.FastGenByArgs(option)
 }
 
+// CollectKeyRangesOption is used to collect key ranges using from input map for given option
+func CollectKeyRangesOption(input map[string]any, collectors ...func(v string)) error {
+	startKeyStr, ok := input["start-key"].(string)
+	if !ok {
+		return errs.ErrOptionNotExist.FastGenByArgs("start-key")
+	}
+	endKeyStr, ok := input["end-key"].(string)
+	if !ok {
+		return errs.ErrOptionNotExist.FastGenByArgs("end-key")
+	}
+	startKeys := strings.Split(startKeyStr, ",")
```

Comment:
startKeyStr could be [a,b,]?

Reply:
TIDB can distribute many partition one time, so the startKeyStr maybe [100,200]